packages/phpstan-rules/packages/nette/src/NodeAnalyzer/UsedLocalComponentNamesResolver.php

```diff
@@ -60,7 +61,10 @@ private function resolveThisGetComponentArguments(Class_ $class): array
                 continue;
             }
 
-            $firstArg = $methodCall->args[0];
+            $firstArg = $methodCall->getArgs()[0] ?? null;
+            if (! $firstArg instanceof Arg) {
+                continue;
```

Comment:
probably return early is the safest way to do on a loop to avoid jump arg:

```suggestion
                return [];
```

Reply:
I'm not sure what you mean. Do you have a failing code sample?

This loop actually goes through all `MethodCalls` found in specific file. So skipping first method call would skip incorrectly all the other method calls.

Reply:
I was thinking about collecting args values, so it is ok then 👍